### PR TITLE
Create AbdulKaderJailani.json

### PR DESCRIPTION
**Pull request by prjailani**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7251"></a>

### CODE REVIEW

File: data/AbdulKaderJailani.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Abdul Kader Jailani P R",
+  "bio": "Web Developer and Digital Marketer at Anavara | CSE'26 | OpenSource Contributor - SSOC'22, SWOC'23, GSSOC'23"
```
Comment (SaraJaoude) [inline data/AbdulKaderJailani.json:3]: add a comma:
```suggestion
  "bio": "Web Developer and Digital Marketer at Anavara | CSE'26 | OpenSource Contributor - SSOC'22, SWOC'23, GSSOC'23",
```

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please make sure your file name matches your GitHub username exactly (it is case sensitive). Instead of `data/AbdulKaderJailani.json` it should be `data/prjailani.json`

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart to help you update this Pull Request.

**Comment by prjailani** [comment]
Now got it @SaraJaoude. Pls, check now. Thanks.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**